ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.0f3-MRTP
+m_EditorVersion: 2017.2.0f3-MRTP3
```

Comment:
@StephenHodgson Not sure about updating the Project version again.  Perhaps we should also remove this file from the  repo.

In any case, this PR shouldn't include changing this file.